packages/db/src/translate/granules.ts

```diff
@@ -46,7 +46,7 @@ export const translateApiGranuleToPostgresGranule = async (
         { name: dynamoRecord.pdrName }
       )
       : undefined,
-    provider_cumulus_id: dynamoRecord.provider ? await providerPgModel.getRecordCumulusId(
+    provider_cumulus_id: (dynamoRecord.provider && dynamoRecord.status !== 'queued') ? await providerPgModel.getRecordCumulusId(
```

Comment:
I'm not sure that I understand what this logic is doing

Reply:
I was getting some failures in the QueueGranules step related to the provider record not being found for the newly created queued granules

Reply:
And got rid of this, not needed